Assets/Scripts/SS3D/Systems/IngameConsoleSystem/ConsolePanelView.cs

```diff
@@ -34,23 +33,35 @@ public class ConsolePanelView : Actor
         private CommandsController _commandsController;
         // Used for choosing command via arrows
         [SerializeField] private List<string> _allPrevCommands = new() {""};
-        private int _chosenPrevCommand = 0;
+        private int _chosenPrevCommand;
         private Controls _controls;
         private Controls.ConsoleActions _consoleControls;
 
+        private void OnEnable()
+        {
+            try
```

Comment:
replace with` if(null && !didstart) return; else { Debug.Error(); return; }` So we can still have an error in case performed is null later in the game.